src/components/TransactionItem.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import * as ReactIcon from "react-icons/ai";
+
+const TransactionItem = ({ transaction, index }) => {
+  let firstClass = index === 0 ? "" : " divide-y divide-gray-300";
+
+  let IconComponent = ({ input }) => {
+    let categoryClass;
+    let icon;
+
+    switch (input) {
+      case "Rent":
+        categoryClass = " bg-indigo-400 text-indigo-800";
+        icon = "AiOutlineHome";
+        break;
+      case "Water & Electricity":
+        categoryClass = " bg-indigo-600";
+        icon = "AiOutlineThunderbolt";
+        // icon = "AiOutlinePoweroff";
+        break;
+      case "Internet":
+        categoryClass = " bg-indigo-200";
+        icon = "AiOutlineWifi";
+        break;
+      case "Food & Groceries":
+        categoryClass = " bg-teal-300 text-teal-800";
+        icon = "AiOutlineShoppingCart";
+        // icon = "AiOutlineShopping";
+        break;
+      case "Subscriptions":
+        categoryClass = " bg-red-200 text-red-800";
+        icon = "AiOutlineSync";
+        // icon = "AiOutlineInteraction";
+        // icon = "AiOutlineEuro";
+        break;
+      case "Other":
+        categoryClass = " bg-orange-200 text-orange-900";
+        // icon = "AiOutlineCreditCard";
+        icon = "AiOutlineShop";
+        break;
+      default:
+        break;
+    }
```

Comment:
Another note on the comments here. If they are not being used then just remove them

Reply:
Noted, thanks! I added a few icon alternatives per expense category for design review. I will remove the extras once we're happy with the final choices 👍